### DISCUSSION TITLE
Wasted my time trying to help my son signup for Global Campus

### DISCUSSION THREAD

**Discussion by mrmikechan** [community/community#49195] [GitHub Education]
Part of the requirement for signing up Global Campus is to create a github account for him.  So I go ahead and work with him to create a new github account.  After creating a github account for him, we are signed into his account.  We then proceed to try and join the Global Campus using is school student ID.  We ended up getting the error message when trying to process the information:

Your GitHub account was too recently created. Please apply again in a few days.  

So how many days should we wait? Still signed in with his credentials, I tried looking at the discussions forum to see if anybody else had this issue.  I didn't see anything and figured I try to post this question.  For the life of me I couldn't figure out where to create a new post.  Well drum roll.... it ends up that if you created a new github account, they have disabled that feature for you.  This is really annoying.  

Does time delaying features for accounts make a difference to whatever you are trying to accomplish on your end other than frustrating new users?

**Comment by luanigithub** [comment]
yes I'm also facing the same error

**Reply by mrmikechan** [reply]
Kind of wonder if the folks over at github realize how frustrating this would be for teenagers trying to sign up and get totally blocked. Well anyways my son is busy with other school activities, so he hasn't bugged me again about this Global Campus.

**Reply by Bjarmah** [reply]
hmm i joined the global campus after about a week since i created my github acc, so try a weeks time then.

**Comment by Bjarmah** [comment]
hmm i joined the global campus after about a week since i created my github acc, so try a weeks time then.

**Reply by MingMingtheDuck** [reply]
Well I just tried again with him using his acct.  At least now he can post from his account.  Anyways he applied again to the global campus and got this error:

Your application cannot be reviewed until you fix the following:
Photo has previously been used. Please upload a new photo.

I know the quickest way is to just take another picture but the usability flow is kind of ridiculous here as a new user is getting stopped out at almost every step lol.

**Reply by MingMingtheDuck** [reply]
Darn thought he was in the clear after uploading a new student id picture.  But now he is getting this error:

Your application cannot be reviewed until you fix the following:
You are significantly more likely to be verified if you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

I have used his full name and school name for company.  Dont know what else to do... so stopped out again.

**Comment by MingMingtheDuck** [comment]
Getting stopped out at proof of academic status:

Your application cannot be reviewed until you fix the following:
You are significantly more likely to be verified if you have completed your [GitHub user profile] with your full name as it appears in your academic affiliation document plus a short bio.

I have used my name and it matches the student id, also used my school for company, added a little blurb for the bio section. I added my city and state plus time zone.

**Comment by Bjarmah** [comment]
Sorry for the late reply, I had some pressure on me due to school, so I did
not visit this acc for a while, but when I filled in my details I also used
my school as my company,  and just placed some basic info about me in my
bio, and it worked.
I honestly have no idea why yours is giving you that error.

**Comment by Bjarmah** [comment]
I also used my school Gmail for the registration (School gmail provided to
all University students), I know of some others who never sent an image of
their student id and only used their student email, but were successfully
verified.

**Comment by Bjarmah** [comment]
In fact, I actually created the acc and got the student package on the same
day[image: image.png]
